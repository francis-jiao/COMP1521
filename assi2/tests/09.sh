# alloc too much space, test3 ignores failed myMalloc()s
# checking that myMalloc() returns NULL on failure
./test3 4000 < data5
