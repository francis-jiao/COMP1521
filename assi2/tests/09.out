+00000 (F, 4096) 
[a] +00008 
+00000 (A, 1008) +01008 (F, 3088) 
[a] +00008 [b] +01016 
+00000 (A, 1008) +01008 (A, 1008) +02016 (F, 2080) 
[a] +00008 [b] +01016 [c] +02024 
+00000 (A, 1008) +01008 (A, 1008) +02016 (A, 1008) +03024 (F, 1072) 
[a] +00008 [b] +01016 [c] +02024 [d] +03032 
+00000 (A, 1008) +01008 (A, 1008) +02016 (A, 1008) +03024 (A, 1008) +04032 (F,   64) 

[a] +00008 [b] +01016 [c] +02024 [d] +03032 
+00000 (A, 1008) +01008 (A, 1008) +02016 (A, 1008) +03024 (A, 1008) +04032 (F,   64) 

+00000 (A, 1008) +01008 (A, 1008) +02016 (A, 1008) +03024 (A, 1008) +04032 (F,   64) 

